### ISSUE TITLE
Illustrations for AnnData sample downloads 

### ISSUE THREAD

**Issue by allyhawkins** [AlexsLemonade/scpca-docs#151]
We need to create a new illustration for the expected download structure when downloading a sample as AnnData.

Below is the expected structure copied from https://github.com/AlexsLemonade/ScPCA-admin/issues/595#issuecomment-1644330881

```
sample_id
├── README.md
├── SCPCL000000_filtered_rna.hdf5
├── SCPCL000000_processed_rna.hdf5
├── SCPCL000000_qc_report.html
├── SCPCL000000_unfiltered_rna.hdf5
└── single_cell_metadata.tsv
```

**Comment by dvenprasad** [comment]
How about this?

<img width="350" alt="anndata-sample-download-folder" src="https://github.com/AlexsLemonade/scpca-docs/assets/15315514/6c9d314d-97a2-4b85-aebf-2e48f3417eae">

**Comment by allyhawkins** [comment]
👍

**Comment by allyhawkins** [comment]
Closed by #161
